source/crud/query/retrieve.txt

```diff
@@ -144,14 +142,112 @@ a ``null`` value if there are no matches.
         ...
       ]
 
-Additional Information
-~~~~~~~~~~~~~~~~~~~~~~
+.. _node-usage-findone:
+
+findOne() Usage Example
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/crud/example-intro.rst
+
+The following full file example finds a single document from the ``movies``
+collection. It uses the following parameters:
+
+- A **query document** that configures the query to return only
+  movies with the title of exactly the text ``'The Room'``.
+
+- A **sort** that organizes matched documents in descending order by
+  rating, so if our query matches multiple documents the returned
+  document will be the document with the highest rating.
+
+- A **projection** that explicitly excludes the ``_id`` field from
+  returned documents and explicitly includes only the ``title`` and
+  ``imdb`` object (and its embedded fields).
```

Comment:
S: style guide - https://www.mongodb.com/docs/meta/style-guide/style/lists/#:~:text=Avoid%20using%20articles%20(a%2C%20an%2C%20the)%20to%20start%20list%20items.

Rephrase these list items to omit articles
ie
```suggestion
- **Filter** that matches documents in which the ``title`` value is ``"The Room"``.

- **Sort** that organizes matched documents in descending order by
  rating, so if our query matches multiple documents the returned
  document will be the document with the highest rating.

- **Projection** that explicitly excludes the ``_id`` field from
  returned documents and explicitly includes only the ``title`` and
  ``imdb`` object (and its embedded fields).
```